opensrp-chw/src/nacp/java/org/smartregister/chw/interactor/ChildHomeVisitInteractorFlv.java

```diff
@@ -453,7 +453,13 @@ protected void evaluateExclusiveBreastFeeding(Map<String, ServiceWrapper> servic
 
         final String serviceIteration = serviceWrapper.getName().substring(serviceWrapper.getName().length() - 1);
```

Comment:
@devcolonel  Retrieveing the serviceIteration works only when the serviceWrapper name contains single digit like "Exclusive breastfeeding5" but not for double digit like "Exclusive breastfeeding11"

Reply:
@maulidkategile fixed